### PR TITLE
Update completedby viewmodel

### PR DESCRIPTION
**Pull request by LayCraft**

### CODE REVIEW

File: embc-app/ViewModels/Registration.cs [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-namespace Gov.Jag.Embc.Public.ViewModels
-{
+namespace Gov.Jag.Embc.Public.ViewModels
+{
     public class RegistrationMappingProfile : Profile
     {
         public RegistrationMappingProfile()
         {
             CreateMap<Registration, Models.Db.EvacueeRegistration>()
                 .ForMember(d => d.EssFileNumber, opts => opts.MapFrom(s => s.EssFileNumber))
```
Comment (ytqsl) [inline embc-app/ViewModels/Registration.cs]: Can be deleted

File: embc-app/ViewModels/Registration.cs [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-                .ForMember(d => d.CompletedById, opts => opts.MapFrom(s => s.CompletedBy.Id))
+                .ForMember(d => d.CompletedById, opts => opts.MapFrom(s => s.CompletedBy))
```
Comment (ytqsl) [inline embc-app/ViewModels/Registration.cs]: Can be deleted

File: embc-app/ViewModels/Registration.cs (hunk 1 above)
Comment (LayCraft) [reply embc-app/ViewModels/Registration.cs]: Line 17?

Comment (ytqsl) [reply embc-app/ViewModels/Registration.cs]: Yes, notice that AutoMapper will map both by convention as the name is the same. Line 18 I'm not sure, you can test with and without and see if it mapped by convention or not

**Pull request merged**